test/e2e/rules_test.go

```diff
@@ -312,3 +312,24 @@ func TestPrometheusRuleApply(t *testing.T) {
 		})
 	}
 }
+
+func TestInvalidRulesAreRejectedOperatorLevel(t *testing.T) {
+	t.Parallel()
+	testCtx := framework.NewTestCtx(t)
+	defer testCtx.Cleanup(t)
+	ns := framework.CreateNamespace(context.Background(), t, testCtx)
+	ruleFilesNamespaceSelector := map[string]string{"excludeFromWebhook": "true"}
+
+	err := framework.AddLabelsToNamespace(context.Background(), ns, ruleFilesNamespaceSelector)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	name := "invalidRule"
+	invalidAlert := "invalidAlert"
+
+	_, err = framework.MakeAndCreateInvalidRule(context.Background(), ns, name, invalidAlert)
+	if err == nil {
+		t.Fatal("Expected invalid prometheusrule to be rejected")
```

Comment:
Hmm the rule should be accepted by the Kube API server if the namespace is excluded from the webhook?